### ISSUE TITLE
Remove LDAP

### ISSUE THREAD

**Issue by skleinfeldt** [isawnyu/pleiades-gazetteer#115]
Really remove it. When add-on is removed there can be LDAP error messages (see standup notes for 4/8/2016).

**Comment by paregorios** [comment]
I think this is done, right @davisagli ?

**Comment by davisagli** [comment]
Nope. We aren't using LDAP any more, but the site still breaks if we remove the plone.app.ldap package.

**Comment by paregorios** [comment]
So I take it that there's still a todo here. In terms of technical debt and angering downstream upgrade deities, how urgent is this?

**Comment by davisagli** [comment]
Not pressing, but it'd be nice to get it cleaned up.

**Comment by paregorios** [comment]
moving to backlog so we can close the "deployment cleanup" milestone
